aliyun/src/main/java/org/apache/iceberg/aliyun/AliyunProperties.java

```diff
@@ -25,21 +25,74 @@
 import org.apache.iceberg.util.PropertyUtil;
 
 public class AliyunProperties implements Serializable {
+  /**
+   * The domain name used to access OSS. OSS uses HTTP Restful APIs to provide services. Different regions are accessed
+   * by using different endpoints. For the same region, access over the internal network or over the Internet also uses
+   * different endpoints. For more information, see:
+   * https://www.alibabacloud.com/help/doc-detail/31837.htm
+   */
+  public static final String OSS_ENDPOINT = "oss.endpoint";
```

Comment:
We generally try to avoid nesting in places that have a small set of specific properties. Is it possible to make these shorter and more standard?

Reply:
As we will introduce the aliyun catalog services named DLF in future, the DLF services also has its endpoint. we need to prefix `oss.` and `dlf.` to  distingush it's an oss endpoint or dlf endpoint.  That's why we use `oss.endpoint` here.